index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="eng">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <Title>Portfolio</Title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+    </head>
+
+    <body>
+      <header>
+        <nav class="header">
+          <a href="#"><h2 class="my-logo">My Logo</h2></a>
+          <img src="./Images/Button.png" alt="Menu">
+        </nav>
+      </header>
+    
+      <main>
+        <section class="headline">
+          <h1>Hey There. I'm Tupai.</h1>
+          <h2>I'm a Software Developer</h2>
+          <p class="about-me">
+            I can help you build a product , feature or website Look through some 
+            of my work and experience! If you like what you see and have a project 
+            you need coded, don't hesitate to contact me.
+          </p>
+
+          <ul class="social">
+            <li><a class ="init-icon" href="https://github.com/Tupai566"><img src="./Images/GitHub.png" alt="GitHub"></a></li>
+            <li><a href="https://www.linkedin.com/in/tupai-mazumder-16109a243/"><img src="./Images/Linkedin.png" alt="Linkedin"></a></li>
+            <li><a href="https://www.facebook.com/tupai.mazumder/"><img src="./Images/V-Hand.png" alt="Facebook"></a></li>
+            <li><a href="https://twitter.com/imtupai566"><img src="./Images/Twitter.png" alt="Twitter"></a></li>
+            <li><a href="https://medium.com/@tupai566"><img src="./Images/M.png" alt="Medium"></a></li>
+          </ul>
```

Comment:
[OPTIONAL]

- [ ] Great job! On using the `ul` & the `li` for your social icons, but currently, the size of the icons are small compared to the Figma design. I recommend matching the Figma design as much as possible to make your project UI better.